### PR TITLE
ci: fix flaky e2e

### LINKED ISSUE #3630: 🤖 - Flaky e2e
### Description

image

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

Closes #3630

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/components/editor/embed/html5/examples/1/index.ts [hunk 1]
```diff
@@ -85,8 +85,12 @@ import {map} from 'rxjs/operators';
             },
         },
     ],
+    host: {
+        class: 'html5-editor-example',
+        '[class._cypress]': 'isCypress',
+    },
+    encapsulation: ViewEncapsulation.None,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/editor/embed/html5/examples/1/index.ts:92]: Maybe it is better to use `::ng-deep` instead ?

File: projects/demo/src/modules/components/editor/embed/html5/examples/1/index.less [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.html5-editor-example._cypress {
+    //noinspection CssInvalidPseudoSelector
+    video::-webkit-media-controls-current-time-display,
+    video::-webkit-media-controls-time-remaining-display,
+    audio::-webkit-media-controls-current-time-display,
+    audio::-webkit-media-controls-time-remaining-display {
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/editor/embed/html5/examples/1/index.less:6]: What do you think about putting all logic inside this file?
https://github.com/Tinkoff/taiga-ui/blob/a0093a010a9263f2880c5e10fafd3df7c76c657a/projects/demo/src/modules/app/cypress.less#L1

```less
._is-cypress-mode .ProseMirror {
    & video: ... {}
}
```

File: projects/demo/src/modules/components/editor/embed/html5/examples/1/index.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/editor/embed/html5/examples/1/index.ts]: fixed

File: projects/demo/src/modules/components/editor/embed/html5/examples/1/index.less (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/editor/embed/html5/examples/1/index.less]: I need to hide not only in .ProseMirror

Comment (splincode) [reply projects/demo/src/modules/components/editor/embed/html5/examples/1/index.less]: In addition, in other situations in the future, who knows, we can change it if necessary

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**